index.html

```diff
@@ -184,5 +264,102 @@ <h4 class="top-banner__subtitle">
         </div>
       </div>
     </section>
+    <section class="why-us">
+      <div class="why-us__container">
+        <div class="why-us__main-content">
+          <h2 class="why-us__title">WHY CHOOSE US</h2>
+          <p class="why-us__subtitle">
+            Problems trying to resolve the conflict between the two major realms
+            of Classical physics: Newtonian mechanics
+          </p>
+        </div>
+        <div class="why-us__grid-container">
```

Comment:
Yes, you have naming issues, but I see you try to do your best. Good for you!

I suggest you to rethink the grid-item namings.
Let's say what kind of data is that? If you take a closer look at the rest of the design you will notice that there are more of these cards. 

So I suggest you to give each of the cards class `card` and use it for all sections in future. 
The parent, the grid one contains cards. So you could give it a class called `cards` or `cards-grid` if you want to specify word "grid" there.
Inside each card you have "card header" and "card content". Here are the class names for them:  `card__header` and `card__content`

Inside header you have icon and title. 
So `header__icn` and `header__title` sound logical for their class names.


Basically you've chosen content agnostic class names as well. And you could use them for future elements. But "grid-item" doesn't tell us what kind of element is there. But "card" tells us that the element is a card and will look like a card. (it's a best practice to call cards elements like these)